src/redux/backend-session.ts

```diff
@@ -224,7 +224,7 @@ const registerMux = (language: Language, dclId: string) => {
   });
 };
 
-const selectImageTag = (language: Language, version: string) => {
+export const selectImageTag = (language: Language, version: string) => {
```

Comment:
Never a good idea to export something just to test it (testing implementation vs testing behaviour).

If it cannot be tested through the public interface, I suggest this refactoring

1. Create folder `src/redux/backend-session`
2. Move `src/redux/backend-session.ts` to `src/redux/backend-session/index.ts`
3. Move the logic to `src/redux/backend-session/selectImageTag.ts`
4. Move your specs to `src/redux/backend-session/selectImageTag.spec.ts`

Reply:
Sure, will do this, thank you!